static/oas/v1/openapi.yml

```diff
@@ -11,6 +11,8 @@ paths:
   # http://localhost:9001/docs#post-/
   /auth:
     $ref: "./auth/auth.yml#/paths/auth"
+  /penaltyTypes:
+    $ref: "./stats/penalty.yml#/paths/penaltyTypes"
```

Comment:
The code is working, but why is `/penaltyTypes` referencing `./stats/penalty.yml` and below we have the same for `/penalties` referencing `./stats/penalties.yml`. Also looking at the folder, there does not exist `./stats/penalty.yml`. Can lines 14 and 15 be removed and the code still work?
Am I seeing something incorrectly?

Reply:
you are totally right on this. penalty.yml does not exist anymore, so I got rid of it. The new path should be under penalties.yml.

Still need those two lines though, cause they represent an api endpoint. The path in it was changed to correct value though